packages/esm-patient-search-app/src/patient-search-page/refine-search/refine-search-tablet.component.tsx

```diff
@@ -0,0 +1,156 @@
+                <SearchField field={genderField} control={control} inTabletOrOverlay={true} isTablet={isTablet} />
+              </div>
+            )}
+            {dobField && (
+              <div className={styles.fieldTabletOrOverlay}>
+                <SearchField field={dobField} control={control} inTabletOrOverlay={true} isTablet={isTablet} />
+              </div>
+            )}
+          </div>
+        )}
+        {otherFields.length > 0 && (
+          <div className={classNames(styles.padded, styles.otherFieldsContainer)}>{otherFieldsRows}</div>
+        )}
+      </>
+    );
+  }, [config, control, isTablet]);
+
+  return (
+    <>
+      <div className={styles.refineSearchBanner}>
+        {!filtersApplied ? (
+          <p className={styles.bodyShort01}>
+            {t('refineSearchTabletBannerText', "Can't find who you're looking for?")}
+          </p>
+        ) : (
+          <div className={styles.refineSearchBannerFilterInfo}>
+            <span className={classNames(styles.filtersAppliedCount, styles.bodyShort01)}>{filtersApplied}</span>{' '}
+            <p className={styles.bodyShort01}>{t('filtersAppliedText', 'search queries added')}</p>
+            <Button kind="ghost" onClick={onResetFields} className={styles.refineSearchDialogOpener} size="sm">
+              {t('clear', 'Clear')}
+            </Button>
+          </div>
+        )}
+        <Button
+          kind="ghost"
+          onClick={onToggleDialog}
+          renderIcon={!showRefineSearchDialog ? ChevronUpIcon : ChevronDownIcon}
+          className={styles.refineSearchDialogOpener}
+          size="sm">
+          {t('refineSearch', 'Refine search')}
+        </Button>
+      </div>
+      {showRefineSearchDialog && (
+        <div className={styles.refineSearchDialogContainer}>
+          <div className={styles.refineSearchDialog}>
+            <div className={styles.refineSearchDialogHeader}>
+              <p className={styles.bodyShort01}>{t('refineSearchHeaderText', 'Add additional search criteria')}</p>
+              <Button
+                kind="ghost"
+                onClick={onToggleDialog}
+                renderIcon={ChevronDownIcon}
+                className={styles.refineSearchDialogOpener}
+                size="sm">
+                {t('refineSearch', 'Refine search')}
+              </Button>
+            </div>
+            <form onSubmit={onSubmit} role="refine-search-tablet">
+              {renderSearchFields}
+              <div className={classNames(styles.buttonSet, styles.paddedButtons)}>
+                <Button kind="secondary" size="xl" onClick={onResetFields} className={styles.button}>
+                  {t('resetFields', 'Reset fields')}
+                </Button>
+                <Button type="submit" kind="primary" size="xl" className={styles.button}>
+                  {t('apply', 'Apply')}{' '}
+                  {filtersApplied ? `(${filtersApplied} ${t('countOfFiltersApplied', 'filters applied')})` : null}
```

Comment:
The `{{count}}` variable in the i18next is treated for pluralization as well.

```suggestion
                  {filtersApplied ? `(${t('countOfFiltersApplied', '{{count}} filters applied', {count: filtersApplied})})` : null}
```